code/modules/clothing/gloves/misc_gloves.dm

```diff
@@ -175,21 +175,28 @@
 /obj/item/clothing/gloves/fingerless/rapid
 	name = "gloves of the North Star"
 	desc = "Just looking at these fills you with an urge to beat the shit out of people."
-	var/accepted_intents = list(INTENT_HARM)
+	var/accepted_intents = list(INTENT_HELP, INTENT_DISARM, INTENT_GRAB, INTENT_HARM)
 	var/click_speed_modifier = CLICK_CD_RAPID
 
 /obj/item/clothing/gloves/fingerless/rapid/Touch(mob/living/target, proximity = TRUE)
+	if(HAS_TRAIT(target, TRAIT_HULK))
+		return FALSE
+
 	var/mob/living/M = loc
```

Comment:
```suggestion
	var/mob/living/M = loc
	
	if(HAS_TRAIT(M, TRAIT_HULK))
		return FALSE

```
This probably worked ok in testing because hulk resets click cooldowns on attack_hulk, so the new cooldown would be instantly overridden, but this allows you to disarm as a hulk quickly and removes the ability to armstrong RP someone with GOTNS

Reply:
Oh I'm dumb, yeah its supposed to be M.